GitExtUtils/Linq/LinqExtensions.cs

```diff
@@ -129,6 +129,14 @@ public static void ForEach<T>([NotNull] this IEnumerable<T> enumerable, [NotNull
             }
         }
 
+        public static void ForEach<T, TResult>([NotNull] this IEnumerable<T> enumerable, [NotNull, InstantHandle] Func<T, TResult> func)
+        {
+            foreach (T t in enumerable)
+            {
+                var unused = func(t);
+            }
+        }
```

Comment:
`\GitUI\Theming\Renderers\ThemeRenderer.cs:61`

```csharp
public void Dispose()
{
    _themeDataHandles.ForEach(NativeMethods.CloseThemeData);
}
```

because `CloseThemeData` returns `int`

Reply:
Is that the only use for that helper method?
If so, please inline it at the callsite.

Reply:
ok